source/faq.txt

```diff
@@ -156,18 +154,19 @@ avoid closing.
 
 One message every 3000 milliseconds is not enough to keep the sockets
 active, so several of the sockets will time out after 5000 milliseconds.
-Reduce the ``poolSize`` in the connection settings to fix the problem.
+Reduce the ``maxPoolSize`` option in the connection settings to lower
+the number of connection the driver can maintain in the connection
+pool.
```

Comment:
Issue:

I like the intent of providing more specificity in what the user should try to accomplish by changing the setting to fix the problem. However, there are a couple minor issues:

- I think "reduce X option" doesn't seem like the right subject-verb agreement. Since it corresponds to the "lower the number of [connections]", the verb only needs to be used once.
- Also, I think "number of connection" should be "number of connections".

Suggestion:

```suggestion
To avoid excessive socket timeouts, lower the number of connections that
the driver can maintain in the connection pool, specified by the ``maxPoolSize`` 
option.
```